### DISCUSSION TITLE
Allow Warps to Cross Bungee network like Portals do.

### DISCUSSION THREAD

**Discussion by RustyDagger** [Zrips/CMI#3068] [Feature Requests]
**Description of feature/improvement request:**
Allow warps location to be set on another bungee server exactly like portals do.
---
### Detailed explanation of what it should do, when and in what situation: ###
make the update position button in the edit warps GUI have a second function if Right clicked.
then it goes into the chat button to set the location like how the portals do it.

### Basic explanation as to why this feature is needed and/or useful to have: ###
Portals are great and all but you have to GO to them and thats inconvenient they also take up a decent bit of space so Im reluctant to have a lot of them. with warps being able to cross the network players will be able to jump across servers and go exactly to where they want ( to other places that are not the default maps spawn like /server takes them. ) currently hiding small 1x1 portals in a dark room and having warps drop players into them. less than ideal very hacky...

**Comment by TheBeast2155** [comment]
i will like it to <3 
i wass on the way to request the same :D
